analyzers/tests/SonarAnalyzer.UnitTest/TestCases/OperatorOverloadsShouldHaveNamedAlternatives.CSharp11.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Threading.Tasks;
+using System;
+
+internal interface IOperator<T> where T : IOperator<T>
+{
+    static virtual T operator ++(T input) => input; // Noncompliant {{Implement alternative method 'Increment' for the operator '++'.}}
+}
+
+public class Foo : IOperator<Foo>
+{
+    public static Foo operator --(Foo a) => a; // Noncompliant
+}
+
+
```

Comment:
Do we have tests for this case already:
```cs
public interface IOperator<T> where T : IOperator<T>
{
    static abstract T operator ++(T input); // Noncompliant
}
```

If not please add a test for this too:
```cs
public interface IPlusOperator<T> where T : IPlusOperator<T>
{
    static abstract T operator ++(T input); // Noncompliant Misses -- operator
}
public interface IPlusMinusOperator<T>: IPlusOperator<T> where T : IPlusMinusOperator<T>
{
    static abstract T operator --(T input); // Compliant ++ operator is derived from IPlusOperator
}
```

Reply:
Please see my comment here: https://github.com/SonarSource/sonar-dotnet/pull/6426#discussion_r1034430503

Reply:
This has been solved. The convo can be resolved.